pages/docs/tutorials/message-validation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Message validation
+description: In this tutorial, you'll learn how to validate AsyncAPI messages (events).
+
+weight: 130
+---
+
+# Introduction
+In this tutorial, you'll learn how to validate messages (events) that are sent to your AsyncAPI application.
+
+# Background context
+Message validation can be performed at both the producer and consumer levels. Message validation requires the participation of the producer, consumer, and broker. We will learn how to validate messages at the consumer level by discarding invalid messages based on the parameters provided.
+
+You will be using the [Eclipse Mosquitto](https://mosquitto.org/) broker. The MQTT protocol provides a lightweight method of messaging using a publish/subscribe model. You will also use an MQTT client that runs an MQTT library and connects to an MQTT broker over a network. Here publishers and subscribers are MQTT clients. The publisher and subscriber labels refer to whether the client is publishing or subscribing to receive messages.
+
+
+In the previous tutorial, you generated your application using the [AsyncAPI Generator](https://github.com/asyncapi/generator) Node.js template. Now you will be validating the messages (events) which you will be sending to your application using a Mosquitto broker and MQTT client.

```

Comment:
since tutorials are supposed to be standalone please add instructions to:
- add instruction to quickly generate asyncapi document 
   ```
   asyncapi new --example=tutorial.yml --no-tty`
   ```
- add instruction to quickly generate server 
   ```
   asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto
   cd output && npm install
   ```

Reply:
done